acceptance_modelisation/toolbox.py

```diff
@@ -27,7 +29,54 @@ def compute_rotation_speed_fov(time_evaluation: Time,
     """
     pointing_altaz = pointing_sky.transform_to(AltAz(obstime=time_evaluation,
                                                      location=observatory_earth_location))
-    omega_earth = OMEGA_EARTH*u.rad
+    omega_earth = OMEGA_EARTH * u.rad
     omega = omega_earth * np.cos(observatory_earth_location.lat) * np.cos(pointing_altaz.az) / np.cos(
         pointing_altaz.alt)
-    return omega
\ No newline at end of file
+    return omega
+
+
+def get_unique_wobble_pointings(observations: Observations, max_angular_separation=0.4):
```

Comment:
Would be better to directly expect an astropy quantity here instead of a float in a predefined unit. It would be in better agreement with the overall philosophy of the code (could also be specified in the declaration).

Reply:
Done